docs/Fedora-Setup.md

```diff
@@ -8,11 +8,6 @@ Modify the steps accordingly for your setup.
 Ensure the following packages are installed
 > $ sudo dnf install qemu-kvm qemu-img git vagrant vagrant-libvirt ansible make libvirt-client
 
-Download and cache vagrant centos7 image.
-```
-$ vagrant box add "https://vagrantcloud.com/centos/7" --provider "libvirt"
-```
-
 For Fedora 33, we have to enable the use_session variable for vagrant libvirt vms to run properly. To do this, create a file ~/.vagrant.d/Vagrantfile with the following content
```

Comment:
I hope this is applicable to Fedora 34 also.

Reply:
yes it is.

Reply:
I hope we can then update the line accrodingly.

Reply:
This was written for and tested with Fedora 33. There is no reason why it shouldn't work for Fedora 34 but I haven't actually tested it with F34. Therefore for now, I would like to keep the docs at F33 as it is the version I tested against.